### PR TITLE
Fix test exclusion logic for nnbd plugins

### PR DESCRIPTION
**Pull request by amirh**
We were skipping tests for all nnbd plugins, see more details in: https://github.com/flutter/flutter/issues/72493

This change should break CI as in the time period we were skipping tests some PRs that break the build landed.
I will land this on red as the tree is actually red (only that we don't see it right now as we're skipping tests)

fixes: https://github.com/flutter/flutter/issues/72493

### CODE REVIEW

**Comment by amirh** [comment]
This seems to be doing the right thing, the master analyze task didn't skip any plugins while the stable one skipped the nnbd packages.

This should land on red (as the tree is actually red right now, this change just makes us see it).

@blasten @bparrishMines I'm going to be OOO for a few weeks, feel free to merge this once it's approved.

**Comment by cyanglaz** [comment]
@blasten Can we land this?

**Comment by blasten** [comment]
I pushed a commit to fix the analyzer issues. Once green, this can be landed.

File: packages/video_player/video_player/example/integration_test/video_player_test.dart
Comment (cyanglaz) [inline packages/video_player/video_player/example/integration_test/video_player_test.dart:5]: nits: revert

**Review by cyanglaz — APPROVED** [review]
LGTM % nits
Assuming ci is fixed

**Pull request merged**
